--- COPYING.orig	2013-05-06 19:39:19.749145318 +0400
+++ COPYING	2013-05-06 19:40:35.510144166 +0400
@@ -2,7 +2,7 @@
 		       Version 2, June 1991
 
  Copyright (C) 1989, 1991 Free Software Foundation, Inc.
-                       59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
  Everyone is permitted to copy and distribute verbatim copies
  of this license document, but changing it is not allowed.
 
--- src/tabwidget.h.orig	2013-05-06 19:49:32.064136010 +0400
+++ src/tabwidget.h	2013-05-06 19:50:31.882135101 +0400
@@ -15,7 +15,7 @@
  *   You should have received a copy of the GNU General Public License     *
  *   along with this program; if not, write to the                         *
  *   Free Software Foundation, Inc.,                                       *
- *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.          *
  ***************************************************************************/
 
 #ifndef TAB_WIDGET
--- src/tabwidget.cpp.orig	2013-05-06 20:06:30.433120531 +0400
+++ src/tabwidget.cpp	2013-05-06 20:06:51.898120204 +0400
@@ -15,7 +15,7 @@
  *   You should have received a copy of the GNU General Public License     *
  *   along with this program; if not, write to the                         *
  *   Free Software Foundation, Inc.,                                       *
- *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.          *
  ***************************************************************************/
 
 #include <QtGui>
--- src/main.cpp.orig	2013-05-06 20:18:27.394109632 +0400
+++ src/main.cpp	2013-05-06 20:18:42.085109409 +0400
@@ -15,7 +15,7 @@
  *   You should have received a copy of the GNU General Public License     *
  *   along with this program; if not, write to the                         *
  *   Free Software Foundation, Inc.,                                       *
- *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.          *
  ***************************************************************************/
 
 #include  <QApplication>
--- src/mainwindow.cpp.orig	2013-05-06 20:19:18.972108848 +0400
+++ src/mainwindow.cpp	2013-05-06 20:19:30.115108679 +0400
@@ -15,7 +15,7 @@
  *   You should have received a copy of the GNU General Public License     *
  *   along with this program; if not, write to the                         *
  *   Free Software Foundation, Inc.,                                       *
- *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.          *
  ***************************************************************************/
 
 #include <QtGui>
--- src/config.h.orig	2013-05-06 20:17:49.398110210 +0400
+++ src/config.h	2013-05-06 20:18:09.914109898 +0400
@@ -15,7 +15,7 @@
  *   You should have received a copy of the GNU General Public License     *
  *   along with this program; if not, write to the                         *
  *   Free Software Foundation, Inc.,                                       *
- *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.          *
  ***************************************************************************/
 
 
--- src/mainwindow.h.orig	2013-05-06 20:18:59.938109138 +0400
+++ src/mainwindow.h	2013-05-06 20:19:12.347108949 +0400
@@ -15,7 +15,7 @@
  *   You should have received a copy of the GNU General Public License     *
  *   along with this program; if not, write to the                         *
  *   Free Software Foundation, Inc.,                                       *
- *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.          *
  ***************************************************************************/
 
 #ifndef MAINWINDOW_H
